test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -262,25 +244,152 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ;
           hello-world-svc (get-service app :HelloWorldService)]
       (is (= (hello-world hello-world-svc) "hello world")))))
 
+(deftest duplicate-service-definitions
+  (testing "Duplicate service definitions causes error with filename and line numbers"
```

Comment:
It shouldn't be treated differently/go down different code paths, but it certainly couldn't hurt, it might not always be that way